developers/smart-contracts/lock-api/README.md

```diff
@@ -28,6 +28,8 @@ function initialize(
 ) external;
 ```
 
+NB: To create non-expiring memberships, just set the `_expirationDuration` to 0.
```

Comment:
This is the only mention of non-expiring memberships I have added. Not sure if it should be expanded in the docs - and if yes where...

Reply:
I think that's good for now, but we need to expose this on the front-end!